packages/mco/components/mco-dashboard/data-policy/cluster-app-card/cluster.tsx

```diff
@@ -356,9 +357,19 @@ export const UtilizationCard: React.FC<UtilizationCardProps> = ({
           clusters={peerClusters}
           title={t('Snapshots synced')}
           queryType={DRDashboard.RBD_SNAPSHOT_SNAPSHOTS}
-          titleToolTip={t(
-            'The y-axis shows the number of snapshots taken. It represents the rate of difference in snapshot creation count during a failover.'
-          )}
+          titleToolTip={
+            <Trans t={t}>
+              <p>
+                This graph provides an overview of the total number of snapshots
+                synced across your clusters.
+              </p>
+              <p className="odf-dashboard-body">
```

Comment:
```suggestion
              <p className="text-muted mco-cluster-app__text--margin-top">
```
incase mco-cluster-app__text--margin-top is giving too much space try to create new class